### PR TITLE
Dev schuenke

### PR DESCRIPTION
**Pull request by schuenke**
Changed the readme and updated the standard config. Maybe it's a good example to get used to PRs and reviews 😁 
@kherz Could you check the reference in the config file?

### CODE REVIEW

**Review by mzaiss — APPROVED** [review]

**Review by mzaiss — COMMENTED** [review]
maybe some units should be added as a inline comment after every line of parameter definition

**Comment by kherz** [comment]
b0 should be 3 T if all the .seq-files are for 3T right now. Standard settings for multiple field strengths (3, 7, 9.4) would be even better, also with adapted water t1 and t2 values

**Comment by mzaiss** [comment]
Field strength in seqs is an important issue: right now the same seq file would yield different flip angles at different field strengths right? and of course wrong offsets as everything is in Hz.

**Comment by kherz** [comment]
> 
> 
> Field strength in seqs is an important issue: right now the same seq file would yield wrong offsets as everything is in Hz.

Yes, that's way they should always include the field strength in the filename. Additionally we can add it as a parameter in the seq definitions

File: sim-library/standard_cest_sim_params.sim [hunk 1]
```diff
@@ -1,90 +1,84 @@
...
-# b0: field strength [T] (float)
-# gamma: gyromagnetic ratio [rad / uT], default is 267.5153 (float)
+###### Scanner settings
+#       b0: field strength [T] (float)
+#    gamma: gyromagnetic ratio [rad / uT], default is 267.5153 (float)
 # b0_inhom: field inhomogeneity [ppm], default is 0.0 (float)
-# rel_b1: relative b1 inhomogeneity, default is 1.0 (float)
-b0: 3
+#   rel_b1: relative amp inhomogeneity, default is 1.0 (float)
+b0: 7
```
Comment (schuenke) [inline sim-library/standard_cest_sim_params.sim:71]:
```suggestion
b0: 3
```

**Review by kherz — APPROVED** [review]

File: sim-library/standard_cest_sim_params.sim [hunk 2]
```diff
@@ -1,90 +1,84 @@
...
-# dw: chemical shift from water [ppm] (float)
-# k: exchange rate [Hz] (float)
-# lineshape: lineshape of the MT Pool as 'Lorentzian', 'SuperLorentzian' or 'None' (str)
+####################################################################################
+# ------ Simulation settings for (py)pulseq-cest BlochMcConnell simulations ------ #
+# ------------------------ https://pulseq-cest.github.io/ ------------------------ #
+####################################################################################
 
-# water properties
+# sources/references for settings in this file:
```
Comment (KerstinHut) [inline sim-library/standard_cest_sim_params.sim:6]: I am missing more info on the simulation background.  I preferred one (or three) lines that recap the main settings (gray matter, 2 cest pools, mt pool) for a quick overview.

**Comment by kherz** [comment]
So, all we need here is the new file name right? _GM_3T_001_bmsim.yaml_?

**Comment by mzaiss** [comment]
Well the question is if 001 will be the standard file or if give this another id like _std

I vote for 001

**Comment by schuenke** [comment]
I agree with Moritz 😮 and vote for 001 👍

**Comment by kherz** [comment]
I would also say a standard file gets an extra name like _std. There might not be a standard for every tissue type and then it can be confusing if GM_..._001 is a standard but necrosis_.._001 is not.

**Comment by mzaiss** [comment]
ok, good point. Im ok with _std but I dont like that its alphabetically last then
suggestions?
__std?

**Comment by KerstinHut** [comment]
I am okay with _std but how about _000 but usual count starts at _001? Double underscore is always a bit tricky, but iif we are the only ones pubishing standards it should be fine.

**Comment by kherz** [comment]
OK, but the name of the standard file is another topic and will take some more time probably. So we can merge this after renaming _standard_cest_sim_params.sim_ to _GM_3T_001_bmsim.yaml_

**Comment by mzaiss** [comment]
yes, works for now.
merge!

**Pull request merged**